### ISSUE TITLE
Failure with repair when bootstrapping a node, but the error message failed to print

### ISSUE THREAD

**Issue by kbr-** [scylladb/scylla#6886]
In this test run:
https://jenkins.scylladb.com/job/scylla-staging/job/abykov/job/feature-cdc-repl-scylla-scylla-test/7/

the test was trying to add 4th node to a 3-node cluster. The operation failed, as we can see in the 4th node's logs:
https://jenkins.scylladb.com/job/scylla-staging/job/abykov/job/feature-cdc-repl-scylla-scylla-test/7/artifact/scylla-cluster-tests/latest/cdc-replication-switch-c-db-cluster-1eeb3332/cdc-replication-switch-c-db-node-1eeb3332-4/system.log

Some relevant fragments of the log:
```
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !INFO    | scylla: [shard 0] storage_service - JOINING: Starting to bootstrap...
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !INFO    | scylla: [shard 0] repair - bootstrap_with_repair: started with keyspaces={system_auth, system_traces, ks1, system_distributed}
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !INFO    | scylla: [shard 0] repair - bootstrap_with_repair: started with keyspace=system_auth, nr_ranges=778
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !INFO    | scylla: [shard 0] repair - repair id [id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc] to sync data for keyspace=system_auth, status=started
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !INFO    | scylla: [shard 0] repair - Repair 1 out of 778 ranges, id=[id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc], shard=0, keyspace=system_auth, table={role_members, roles}, range=(9124966397736039236, 9127268275554238495]
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !INFO    | scylla: [shard 0] repair - Repair 2 out of 778 ranges, id=[id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc], shard=0, keyspace=system_auth, table={role_members, roles}, range=(9116908935233387255, 9124966397736039236]
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !INFO    | scylla: [shard 0] repair - Repair 3 out of 778 ranges, id=[id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc], shard=0, keyspace=system_auth, table={role_members, roles}, range=(9095165125922840312, 9116908935233387255]
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !ERR     | scylla: [shard 0] repair - Repair 3 out of 778 ranges, id=[id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc], shard=0, keyspace=system_auth, table={role_members, roles}, range=(9095165125922840312, 9116908935233387255], peers={10.0.171.53}, live_peers={}, status=failed: mandatory neighbor=10.0.171.53 is not alive
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !INFO    | scylla: [shard 1] repair - Repair 1 out of 778 ranges, id=[id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc], shard=1, keyspace=system_auth, table={role_members, roles}, range=(9124966397736039236, 9127268275554238495]
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !INFO    | scylla: [shard 1] repair - Repair 2 out of 778 ranges, id=[id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc], shard=1, keyspace=system_auth, table={role_members, roles}, range=(9116908935233387255, 9124966397736039236]
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !INFO    | scylla: [shard 1] repair - Repair 3 out of 778 ranges, id=[id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc], shard=1, keyspace=system_auth, table={role_members, roles}, range=(9095165125922840312, 9116908935233387255]
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !ERR     | scylla: [shard 1] repair - Repair 3 out of 778 ranges, id=[id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc], shard=1, keyspace=system_auth, table={role_members, roles}, range=(9095165125922840312, 9116908935233387255], peers={10.0.171.53}, live_peers={}, status=failed: mandatory neighbor=10.0.171.53 is not alive
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !WARNING | scylla: [shard 0] repair - repair id [id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc] on shard 0 failed: fmt::v6::format_error (invalid type specifier)
2020-07-21T06:45:54+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !WARNING | scylla: [shard 1] repair - repair id [id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc] on shard 1 failed: fmt::v6::format_error (invalid type specifier)
2020-07-21T06:45:54+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !WARNING | scylla: [shard 0] repair - repair id [id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc] failed: std::runtime_error ({shard 0: fmt::v6::format_error (invalid type specifier), shard 1: fmt::v6::format_error (invalid type specifier)})
2020-07-21T06:45:54+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !WARNING | scylla: [shard 0] repair - repair id [id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc] to sync data for keyspace=system_auth, status=failed: std::runtime_error ({shard 0: fmt::v6::format_error (invalid type specifier), shard 1: fmt::v6::format_error (invalid type specifier)})
2020-07-21T06:45:54+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !INFO    | scylla: [shard 0] init - Shutting down gossiping
2020-07-21T06:45:54+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !INFO    | scylla: [shard 0] gossip - My status = BOOT
2020-07-21T06:45:54+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !WARNING | scylla: [shard 0] gossip - No local state or state is in silent shutdown, not announcing shutdown
2020-07-21T06:45:54+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !INFO    | scylla: [shard 0] gossip - Disable and wait for gossip loop started
```

So it looks like repair failed for some reason. But it's unclear what it is:
```
2020-07-21T06:45:53+00:00  cdc-replication-switch-c-db-node-1eeb3332-4 !WARNING | scylla: [shard 0] repair - repair id [id=1, uuid=d3823b7b-13bd-4923-9998-1d16406b7ebc] on shard 0 failed: fmt::v6::format_error (invalid type specifier)
```

The message didn't get printed.

I guess we should try to fix the error message first and then try to reproduce.

Adding CDC label, since the error happened in the context of CDC.

**Comment by slivne** [comment]
@asias you fixed 28f8798464f993727cd97d70ecc284d62cfe36d7 is this the same ?

**Comment by asias** [comment]
Yes. It is a duplication of #6874.
